sdk/core/azure-core/src/private/environment_log_level_listener.hpp

```diff
@@ -12,8 +12,6 @@
 #if !defined(NOMINMAX)
 #define NOMINMAX
 #endif
-
-#include <windows.h>
```

Comment:
@antkmsft is there any reason for us to have this header included here? Same with the `WIN32_LEAN_AND_MEAN`/etc. above.

Reply:
Yes - so it is only available on non-UWP Windows, and to detect that, we need windows.h